### DISCUSSION TITLE
What is suspense mode ?

### DISCUSSION THREAD

**Discussion by devianyeth** [wevm/wagmi#1244] [Q&A]
Wanted to know what is this suspense mode? mentioned here https://wagmi.sh/docs/prepare-hooks/usePrepareContractWrite#suspense-optional

**Comment by Fabien-Morrow** [comment]
Same, what is this ?

**Comment by tmm** [comment]
Experimental React Suspense support based on [React Query's implementation](https://tanstack.com/query/v4/docs/guides/suspense)
